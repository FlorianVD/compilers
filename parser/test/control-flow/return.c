int returns()
{
    return;
    return 1 + 2;
}
