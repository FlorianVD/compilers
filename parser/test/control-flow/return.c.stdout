└── Program
    └── FuncDecl: returnType = 'int', name = 'returns'
        └── CompoundStmt
            ├── ReturnStmt
            └── ReturnStmt
                └── BinaryOpExpr: op = '+'
                    ├── IntLiteral: value = '1'
                    └── IntLiteral: value = '2'
